### ISSUE TITLE
The GetImage method returned signature on the left most in the image

### ISSUE THREAD

**Issue by slam023** [xamarin/SignaturePad#87]
I am not sure is this a bug or intention to have it this way. On iOS, I have a rectangle signaturePad. When I signed either in the center or on the right side, the image returned the signature drawn on the left side instead of leaving it the same location at the time I drawn

**Comment by mattleibow** [comment]
This may have something to do with a bug I just fixed... This bug occurred when the signature was incorrectly positioned when the image was not cropped.

Could you attach a screenshot of the signature and the image so I can confirm that this is the bug?

**Comment by slam023** [comment]
<img width="500" alt="fa222edf-393c-4bd9-bc4a-836b60559640" src="https://user-images.githubusercontent.com/16716114/29193821-0e1ebbda-7e7b-11e7-9682-19061bad1c90.png">

The yahoo initially drawn on the center but the image displayed the signature on the left side.

**Comment by mattleibow** [comment]
What happens when you set the cropping flag to true? It should crop - I just want to make sure. It looks like this is the case.

**Comment by mattleibow** [comment]
Th fix should come out with the v2.2 NuGet (not yet released)

**Comment by slam023** [comment]
I realised another thing not sure is it right or not. I still using the same version. On android, if I set should crop to true, the signature didn't get resized to a given size I passed in and get cropped. If I set to false, the signature will be within the image which I was wanted. On iOS, if I set  should crop to false, the signature get cropped, whereas if Should Crop is true, the signature will within the image but on the left size. 
Is it part of the fixed you have done.

**Comment by Smaths** [comment]
I'm also encountering this unexpected behavior. When calling `GetImage()` on an iOS device, the drawing/signature is cropped despite the explicitly setting `ShouldCrop = false`. I've tried the following sets of code without success. 

**Set 1**
```
ImageConstructionSettings imageSettings = new ImageConstructionSettings
{
    ShouldCrop = false
};
UIImage image = _signatureView.GetImage(imageSettings);
```

**Set 2**
```
UIImage annotationImage = _signatureView.GetImage(false);
```

**Comment by Sean-LINKSOFT** [comment]
Just wondering when v2.2 will be released to solve this issue?

**Comment by Smaths** [comment]
In case anyone is in need of a quick and quite dirty hack. I needed this working for a demo tomorrow and had to come up with something. 

The hack is basically to create a stroke the width of the signature frame. Then load this stroke into the signatureView and hide it by placing it's x coordinate below zero (ex. -1). This will prevent the signatureView from cropping itself and placing the x coordinate below zero value will hide the line from appearing. This isn't a great solution but will get you somewhat reliable results until the fix comes out.

**Comment by mattleibow** [comment]
This should be fixed in v2.2. Please re-open if the issue still exists when the package is published.
